lib/schema.js

```diff
@@ -343,3 +343,8 @@ export const toSchema = (browser, options = {}) => {
 
   return schema;
 };
+
+export const getAllSchemas = () => {
+    const schemaUris = Object.keys(schemaRegistry);
+    return schemaUris;
+}
```

Comment:
This can be simplified.

```suggestion
export const getAllSchemas = () => Object.keys(schemaRegistry);
```